frame/App/Body/Main/index.js

```diff
@@ -80,7 +80,6 @@ const Main = (): React.Node => {
       });
     }
   }, [user, renderedLists]);
-
```

Comment:
More like code standards, it looks better if you keep a newline before return statements right? I think it's more readable but what do you think?